docs/conf/portland/2023/venue.rst

```diff
@@ -0,0 +1,8 @@
+:template: {{year}}/generic.html
+
+About the Venue
+===============
+
+Our venue this year will be online!
+We're currently working to figure out the exact platform we'll be using.
+This page will be updated once we figure it out.
```

Comment:
Need to update this.

Reply:
Yes, this isn't linked yet though, because we only run a landing page.